validateFileNames.js

```diff
@@ -67,23 +75,28 @@ if (filesInViolation.length > 0) {
   shelljs.exit(1);
 } else {
   // No, exit with 0
+  console.info('All checked files are camelCased');
   shelljs.exit(0);
 }
 
 function getFiles() {
   // Let's only validate files managed by git
-  const files = shelljs.exec('git ls-files', { silent: true });
-
-  return files.split('\n').filter(file => {
-    // remove empty strings from the array and remove files in ignored paths
-    return file.length !== 0 && !isIgnored(file);
-  });
+  const { code, stdout } = shelljs.exec('git ls-files', { silent: true });
```

Comment:
Well, okay. But I'm positive that the earlier implementation was working.

Perhaps it's a monkey patched string that's created with `new String('whatever')`

`typeof new String('whatever')` is an `object`.